tests/test_quick_run_config_generator.py

```diff
@@ -911,6 +930,138 @@ def _get_next_run_config_ensemble(self,
 
         self.assertEqual(perf_config['batch-size'], 1)
 
+    def _get_next_run_config_bls(self, max_concurrency=0, min_concurrency=0):
+        """
+        Test that get_next_run_config() creates a proper RunConfig for BLS
```

Comment:
I love this test if it does everything it says it does! Very thorough

Reply:
It does! And, thanks.